Sources/BraintreePayPal/BTPayPalAnalytics.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class BTPayPalAnalytics {
+    static let vaultRequestStarted = "paypal:vault-tokenize:started"
+    static let checkoutRequestStarted = "paypal:checkout-tokenize:started"
```

Comment:
Do we need the distinction between these two requests in the analytic events at all?

Reply:
Yeh this is a great callout. PayPal has 2 tokenize methods, vault & checkout.

With the current implementation - there is no way to tie the end (failed, success, cancel) events to the different vault or checkout start events.

So we could either:
* Not care about conversion per PP flow type, and just do all generic `tokenize`
* Add distinct start and end events for `tokenize-vault` and `tokenize-checkout`

Another thing we _could_ do that might just be muddy - send additional information on the integration type in the metadata.

Reply:
We could check in with Arpan to see his recommendation, but I'm 99% confident he'll want separate vault & checkout metrics.

Reply:
Is it sufficient to have the start events to specify the different flows?

Reply:
Also regardless, if we do go that route - it should be a follow-up ticket. This PR I think we are OK for now with separate vault & checkout events.

Reply:
So if we are going for the meta-data for the payment types, should I just keep one requestStarted message?

Reply:
So just to clarify, do you want me to have the integration type, or payment type, in all of the succeeded, failed and canceled messages or is it sufficient to have it in the start method for this ticket? If the latter is true, I'd need either the helper method that inserts the paymentType or static function in the BTPayPalAnalytics that takes in payment type

Reply:
👋 Yup - that's a great call, that we will need to inject the param value in the `sendAnalyticsEvent()` call if we go that route. 

I think we should first decide as a team if we want to include "additional info" in the metadata, or not. I think starting a thread in #mobile-sdk-core would be good, since this involves Android too.

And then depending on the conclusion, we can spin up JIRA tickets for the work.

How does that sound?

Reply:
But for this PR, I think we are OK not to worry about additional metadata info, and can include "vault" and "checkout" in event names until we decide on a plan and make JIRAs